server/src/db/migrations/20210210143402_createBooks.cjs

```diff
@@ -10,6 +10,7 @@ exports.up = async (knex) => {
     table.bigIncrements("id").primary();
     table.string("title").notNullable();
     table.string("author");
+    table.bigInteger("bookListId").unsigned().nullable().index().references("bookLists.id");
```

Comment:
noooooo you shouldnt need to do this because you have join table between books and bookLists